pages/fiscal-sponsorship/apply/index.js

```diff
@@ -101,6 +101,7 @@ export default function Apply() {
             </Heading>
           </Box>
           <HCBInfo />
+          <ContactBanner sx={{ borderRadius: 'default', bg: 'snow' }} />
```

Comment:
On large monitors, this grows a bit too wide.

<table>
  <tr>
    <th>Before
    <th>After
  <tr>
    <td><img src="https://github.com/hackclub/site/assets/20099646/de675216-f425-4640-8af0-2ce724bf2d25" />
    <td><img src="https://github.com/hackclub/site/assets/20099646/ed0401e9-7af4-4474-b957-c09afdadedea" />

</table>

```suggestion
          <ContactBanner sx={{ borderRadius: 'default', bg: 'snow', width: 'fit-content' }} />
```